src/main/java/upbrella/be/store/dto/response/SingleStoreResponse.java

```diff
@@ -20,5 +23,25 @@ public class SingleStoreResponse {
     private String instagramId;
     private double latitude;
     private double longitude;
+    private String content;
     private List<String> imageUrls;
+
+    @QueryProjection
+    public SingleStoreResponse(long id, String name, String category, SingleClassificationResponse classification, SingleSubClassificationResponse subClassification, boolean activateStatus, String address, String umbrellaLocation, String businessHours, String contactNumber, String instagramId, double latitude, double longitude, String content, List<String> imageUrls) {
+        this.id = id;
+        this.name = name;
+        this.category = category;
+        this.classification = classification;
+        this.subClassification = subClassification;
+        this.activateStatus = activateStatus;
+        this.address = address;
+        this.umbrellaLocation = umbrellaLocation;
+        this.businessHours = businessHours;
+        this.contactNumber = contactNumber;
+        this.instagramId = instagramId;
+        this.latitude = latitude;
+        this.longitude = longitude;
+        this.content = content;
+        this.imageUrls = imageUrls;
+    }
```

Comment:
파라미터를 이렇게 하나 하나 다 받아야 하는 이유가 있나요? 
예를 들어서 storeMeta에 address와 content가 있다면 아래와 같이 바꿀 수는 없나요?

```suggestion
    @QueryProjection
    public SingleStoreResponse(StoreMeta storeMeta) {
        this.address = storeMeta.getAddress();
        this.content = storeMeta.getContent();
    }
```

Reply:
쿼리dsl에서 사용하는 거라서 필드를 다 받아야 하는 것으로 알고 있습니다. 
Entity에도 QueryProjection용 생성자를 만들면 가능하긴 한데, 이것에 대한 회의는 진행해봐야 할것 같습니다.